Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -57,14 +63,18 @@ public struct OfflineMapAreasView: View {
                 .textCase(nil)
             }
             .task {
-                await makePreplannedMapModels()
+                await mapViewModel.requestUserNotificationAuthorization()
             }
             .task {
-                await mapViewModel.requestUserNotificationAuthorization()
+                await makePreplannedMapModels()
+                isConnected = networkMonitor.isConnected
+            }
+            .task(id: networkMonitor.isConnected) {
+                offlineBannerIsPresented = !networkMonitor.isConnected
             }
```

Comment:
This can be synchronous

```swift
            .onChange(of: networkMonitor.isConnected) { _ in
                offlineBannerIsPresented = !networkMonitor.isConnected
            }
```